src/tests/end-to-end/common/browser.ts

```diff
@@ -12,7 +12,7 @@ export class Browser {
     private memoizedBackgroundPage: Page;
     private pages: Array<Page> = [];
 
-    constructor(private readonly underlyingBrowser: Puppeteer.Browser) {
+    constructor(public readonly browserInstanceId: string, private readonly underlyingBrowser: Puppeteer.Browser) {
```

Comment:
Just curious, what are the reasons for making `browserInstanceId` public? I can't find an instance where this member variable is accessed outside the `Browser` class.

Reply:
Oops, that was leftover from when I was using a different design for the onPageCrash handler. You're right, this can be private :)